core/Cargo.toml

```diff
@@ -245,9 +245,7 @@ futures = { version = "0.3", default-features = false, features = [
 http = "1.1"
 log = "0.4"
 md-5 = "0.10"
-# TODO: remove once_cell when lazy_lock is stable: https://doc.rust-lang.org/std/cell/struct.LazyCell.html
 ghac = { version = "0.2.0", optional = true }
-once_cell = "1"
```

Comment:
The discussion of bumping opendal MSRV is ongoing: https://github.com/apache/opendal/discussions/5864

Reply:
seen that and will draft this waiting for MSRV update

Reply:
closing